model/metrics/hardware/common.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  # COMMON METRICS
+  - id: metric.hardware.attributes
+    type: attribute_group
+    brief: "Attributes for hardware metrics"
+    extends: hardware.attributes.common
+    attributes:
+      - ref: hw.type
+        requirement_level: required
+
+  - id: metric.hardware.energy
+    type: metric
+    metric_name: hw.energy
```

Comment:
```suggestion
    metric_name: hw.energy.consumption
```
?

Reply:
I don't change in this PR anything, I do only refactoring (in cases where it's possible), if we would like to change names and introduce a breaking change - this should be done later for simplicity

Reply:
@lmolkova This can be discussed in a separate issue. The `.consumption` suffix is not necessary. Also, this counter may also cover energy production (batteries, for example), in which case we will use an attribute to indicate the *direction* (`direction="out|in"`).

Reply:
created https://github.com/open-telemetry/semantic-conventions/issues/1071 to follow up